### ISSUE TITLE
raising bounties

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#35]
questions:
* requirements ?
  * e.g. opensourced stuff, with a good licence for us
* how much ?

some ideas: category: [bounty](https://github.com/Erkan-Yilmaz/Gridcoin-tasks/labels/bounty)

mentioned in [hangout 22](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2017_01_21.MD):

**Comment by grctest** [comment]
>requirements for the product
Yeah, if development is being funded by the foundation it must be open source. We shouldn't subsidize companies.

**Comment by Erkan-Yilmaz** [comment]
> how much for each bounty

there is now a 10000 GRC bounty for making a secure, opensourced Gridcoin pool: #2

**Comment by Peppernrino** [comment]
i was also thinking it would be cool if we could fund a pool of devs to make some working linux GPU drivers...

**Comment by Erkan-Yilmaz** [comment]
mentioned in [hangout 26](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2017_03_18.MD): making a bounty for removing/improving "team requirement"
